server-spi-private/src/main/java/org/keycloak/utils/OCSPProvider.java

```diff
@@ -126,13 +124,6 @@ protected byte[] getEncodedOCSPResponse(KeycloakSession session, byte[] encodedO
         CloseableHttpClient httpClient = session.getProvider(HttpClientProvider.class).getHttpClient();
         HttpPost post = new HttpPost(responderUri);
         post.setHeader(HttpHeaders.CONTENT_TYPE, "application/ocsp-request");
-
-        final RequestConfig params = RequestConfig.custom()
```

Comment:
This change makes sense, but strictly speaking, it is background incompatible change (as for example default socket timeout is 5 seconds for the default HTTP client provider).

IMO it is OK to include this change in your PR, but it would be also good to add a note about this to the file https://github.com/keycloak/keycloak/blob/main/docs/documentation/upgrading/topics/changes/changes-26_5_0.adoc . Probably to the "Notable changes" section. I assume that note can mention that OCSP behaviour used hardcoded socket and connection timeouts of 10 seconds before, but now it is updated to use the settings from the HTTP client configuration.

WDYT?